controllers/app-controllers.js

```diff
@@ -14,3 +15,13 @@ exports.getArticle = (req, res, next) => {
     })
     .catch(next)
 }
+
+exports.updateVotes = (req, res, next) => {
+    const {article_id} = req.params
+    const addVotes= req.body.inc_votes 
```

Comment:
(subjective)

Try to have some more consistency with when you are destructuring values